asset/cloud-client/quickstart_batchgetassetshistory.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+
+def batch_get_assets_history(project_id, asset_names):
+    # [START asset_quickstart_batch_get_assets_history]
+    from google.cloud import asset_v1beta1
+    from google.cloud.asset_v1beta1.proto import assets_pb2
+    from google.cloud.asset_v1beta1 import enums
+
+    # TODO project_id = 'Your Google Cloud Project ID'
+    # TODO asset_names = 'Your asset names list, e.g.:
+    # ["//storage.googleapis.com/[BUCKET_NAME]",]'
+
+    client = asset_v1beta1.AssetServiceClient()
+    parent = client.project_path(project_id)
+    content_type = enums.ContentType.RESOURCE
+    read_time_window = assets_pb2.TimeWindow()
+    read_time_window.start_time.GetCurrentTime()
+    response = client.batch_get_assets_history(
+        parent, content_type, read_time_window, asset_names)
+    print(response)
```

Comment:
It may be better explicitly print the response members using `print('Member: {}'.format(reponse.member))`

Reply:
Committed with the suggested print. Thanks!

Reply:
> I actually didn't mean member, I meant the relevant member that is included in the response and that we want to show here.
I see... Updated the code to print response.assets. Thanks.